### PR TITLE
Added documentation for installing Command Line Tools

### PR DESCRIPTION
**Pull request by carolinekraus02**
I added documentation describing how to install automake, libtool, cmake, pkg-config, wget, and meson and included some common pitfalls and solutions for those processes. This will improve contributors' understanding because the installation prerequisites are more explicitly explained.

### CODE REVIEW

**Review by t-paul — APPROVED** [review]
Thanks! Looks good to me, but I have not touched macOS / Homebrew for a long time. @kintel?

File: README.md [hunk 1]
```diff
@@ -141,8 +141,38 @@ To pull the various submodules (incl. the [MCAD library](https://github.com/open
 Prerequisites:
 
 * Xcode
+Terminal tools
```
Comment (kintel) [inline README.md:144]: -> Xcode command-line tools

File: README.md [hunk 2]
```diff
@@ -141,8 +141,38 @@ To pull the various submodules (incl. the [MCAD library](https://github.com/open
...
+* In terminal, run: brew help
+* Run: brew -v
+* If neither command returns information, this means Homewbrew is not installed
+* Proceed in terminal and run: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+* If more help is needed, visit https://stackoverflow.com/questions/21577968/how-to-tell-if-homebrew-is-installed-on-mac-os-x
+automake
+* In terminal, run: brew install automake
+libtool
+* In terminal, run: brew install libtool
+* May have to use "glibtool" instead on some newer versions of MacOSX
```
Comment (kintel) [inline README.md:158]: Could you explain this a bit better? On what version of macOS is `glibtool` available on homebrew?

File: README.md [hunk 3]
```diff
@@ -141,8 +141,38 @@ To pull the various submodules (incl. the [MCAD library](https://github.com/open
...
+cmake
+* Ensure no old versions are lingering by running in terminal: sudo apt purge --auto-remove cmake
+* Visit http://www.cmake.org/download/ and download the appropriate package for your operating system and version
+* Install using: sudo mkdir /opt/cmake
+* Finish install; sudo sh <installer filename> --prefix=/opt/cmake
+* Common issues with cmake: version too outdated or filepaths contain unecessary spaces
+* If issues persist, visit this resource https://hsf-training.github.io/hsf-training-cmake-webpage/07-commonproblems/index.html
```
Comment (kintel) [inline README.md:165]: The cmake instructions seem to be a mixture or macOS and Linux instructions. Why would we use `apt purge` on macOS?

File: README.md [hunk 4]
```diff
@@ -141,8 +141,38 @@ To pull the various submodules (incl. the [MCAD library](https://github.com/open
...
+wget
+* In terminal, run: brew install wget
+meson
+* In terminal, run: brew install meson
+* Common error returned: "error cannot open file '_____.lib'"
+* To address this, make sure the proper language is installed
+* Ex.: for C++ projects, install Visual C++ for Desktop Development
+* In Visual Studio: Installed > Change Installed VS CE > Workloads > Desktop Development with C++ > apply and save
```
Comment (kintel) [inline README.md:173]: Why mention Visual Studio in macOS install instructions?

**Comment by kintel** [comment]
Added some comments - the instructions are an odd mix of using Linux, Windows and macOS tools to install dependencies on macOS. Needs to either be cleaned up, or split up into sections for each OS.

**Comment by kintel** [comment]
Closing due to lack of feedback

**Pull request closed without merging**